IanGRap/Squish
# Fix ui prefab bug

Contributors: Alfredo Cuevas

Goal: Fix a weird issue where the UI Manager prefab could not be updated. 

Important files changed: 
UIScoresMenu.cs: I tried to ensure that the index in the for loop was never greater than the length of both the scores array and active players list. 

* Note: I didn't actually update the UI Manager prefab to ensure that I didn't cause any merge conflicts. 

How to test: 
Update the UIManager prefab
If the UI Manager prefab is updated the score UI should still work properly from both the Player select and regular game play scenes.
